docs/source/getting-started/installation.rst

```diff
@@ -59,39 +47,11 @@ Next steps
 Alternative: manual download
 ****************************
 
-The cURL command above will automatically download and run the DAML installation script from GitHub (using TLS). If you require a higher level of security, you can instead install the SDK by manually downloading the compressed tarball, verifying its signature, extracting it and manually running the install script.
-
-Note that the Windows installer is already signed (within the binary itself), and that signature is checked by Windows before starting it. Nevertheless, you can still follow the steps below to check its external signature file.
-
-To do that:
+If you're extra careful about security, you can look at :doc:`our detailed instructions for manual download and installation <manual-download>`.
```

Comment:
```suggestion
If you want to verify the SDK download for security purposes before installing, you can look at :doc:`our detailed instructions for manual download and installation <manual-download>`.
```
Might just be me but it sounded a bit too casual for my tastes.

Reply:
You're right. I was trying to not give the impression that the above was insecure. Though I guess it somewhat is. 🤷